tests/product/test_create_images.py

```diff
@@ -0,0 +1,98 @@
+from nose.plugins.attrib import attr
+from subprocess import check_output
+from sys import argv
+from tests.no_hadoop_bare_image_provider import NoHadoopBareImageProvider
+from tests.product.base_product_case import BaseProductTestCase
+
+import unittest
+
+
+@attr('quarantine')
+class TestSetupImages(BaseProductTestCase):
```

Comment:
I see what you did here, it's pretty clever :) but a little hacky in my opinion. You call into `setup_cluster()` and piggyback on that infrastructure. However, the price you pay is that your methods need to be tests. I think we need to rip out `check_cluster()` from `BaseProductTestCase` so that this module can be completely standalone and doesn't need to run as a fake test.